docs/specs/Mountain State Adventure Destination/SPEC-13-template-lake/architecture/component-diagram.md

```
@@ -0,0 +1,488 @@
+        │                                              │                │
+        │                                    ┌─────────▼──────┐  ┌─────▼────────┐
+        │                                    │ Zod Schemas    │  │ Props        │
+        │                                    │ Validation     │  │ Destructure  │
+        │                                    └────────────────┘  └──────────────┘
+        │
+        └─── Sections (11 total) ───────────────────────────────────────────┐
+                                                                            │
+                                                                            │
+        ┌───────────────────────────────────────────────────────────────────┘
+        │
+        │
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)